apache/superset
# fix(dataset): Page blanks on large data load

### SUMMARY
Set the pagination to `true` to prevent chart page from crashing due to heavy data load.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Before:
![No Pagination 2](https://user-images.githubusercontent.com/26679866/101644013-01a1f100-3a35-11eb-9c44-112eca777776.gif)

After:
![With Pagination 2](https://user-images.githubusercontent.com/26679866/101646492-e684b080-3a37-11eb-866f-f6d0df79c0c7.gif)

### TEST PLAN
1. Navigate to **Datasets** page
2. Select **wb_health_population**
3. Set the **Visualization Type** to **Linear Chart**
4. Set the **Time Range** to **No Filter**
5. Run the query
6. Under the **Data** panel select **View Samples** tab

Data should load in couple of seconds.

### ADDITIONAL INFORMATION
